src/pages/plugins-2023/index.md.ejs

```diff
@@ -2,7 +2,7 @@
 layout: content.html
 ---
 
-<%- include('plugins') %>
+<%- include('plugins_bak') %>
```

Comment:
why the _bak prefix?

Reply:
It was the "backup" of the previous manually-created data, using npm and the previous data, but I'm now using it as the primary source for the page so as to only have npm-published plugins visible - I'll rename it